drs_downloader/cli.py

```diff
@@ -1,11 +1,183 @@
+import logging
+from pathlib import Path
+from typing import List
+import math
+import tqdm
 import click
+import csv
 
+from drs_downloader.clients.gen3 import Gen3DrsClient
+from drs_downloader.clients.mock import MockDrsClient
+from drs_downloader.clients.terra import TerraDrsClient
+from drs_downloader.manager import DrsAsyncManager
 
-@click.command()
+from drs_downloader import DEFAULT_MAX_SIMULTANEOUS_OBJECT_SIGNERS
+
+logging.basicConfig(format='%(asctime)s %(message)s',  encoding='utf-8', level=logging.INFO)
+logger = logging.getLogger(__name__)  # these control our simulation
+
+
+@click.group()
 def cli():
-    """Welcome to drs_downloader."""
-    print(cli.__doc__)
+    """Copy DRS objects from the cloud to your local system ."""
+    pass
+
+
+@cli.command()
+@click.option("--silent", "-s", is_flag=True, show_default=True, default=False, help="Display nothing.")
+@click.option("--destination_dir", "-d", show_default=True, default='/tmp/testing',
```

Comment:
I think the separator in option identifiers should be changed from `_` to `-`.

I didn't see the specified one way or the other in the [specification](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html) I looked at (may have overlooked it).

Yet, it is the common convention for all the CLI commands I recall using, for example, the GNU/Linux `cp` command ([here](https://man7.org/linux/man-pages/man1/cp.1.html)).

Reply:
Agreed.  Will address today in a change to this PR, including test.

Reply:
Fixed and tested here: https://github.com/anvilproject/drs_downloader/blob/feature/manager/tests/integration/test_problems.py#L11-L17